src/commoncode/testcase.py

```diff
@@ -183,7 +183,7 @@ def get_temp_dir(self, sub_dir_path=None):
             test_tmp_root_dir = os.path.join(scancode_root_dir, 'tmp')
             # not we add a space in the path for testing path with spaces
             test_run_temp_dir = fileutils.get_temp_dir(
-                base_dir=test_tmp_root_dir, prefix='scancode-tk-tests -')
+                base_dir=test_tmp_root_dir, prefix='scancode-tk-tests-')
```

Comment:
This is a small space that is there for a reason: to ensure that we run tests on paths that contain at least one space, so do you mind reverting this?

Reply:
Updated. I didn't notice the comment one line above!